static/templates/settings/account-settings.handlebars

```diff
@@ -56,9 +56,25 @@
         <input type="file" name="user_avatar_file_input" class="notvisible" id="user_avatar_file_input" value="{{t 'Upload avatar' }}" />
         <div id="upload_avatar_spinner"></div>
         <button class="button standalone" id="user_avatar_upload_button">{{t 'Customize avatar' }}</button>
+        <button class="button standalone btn-danger" id="user_deactivate_account_button">{{t 'Deactivate Account' }}</button>
       </div>
 
     </form>
 
+      <div id="deactivation_user_modal" class="modal hide fade" tabindex="-1" role="dialog" aria-labelledby="deactivation_user_modal_label" aria-hidden="true">
+        <div class="modal-header">
+          <button type="button" class="close" data-dismiss="modal" aria-hidden="true">×</button>
+          <h3 id="deactivation_user_modal_label">{{t "Deactivate Your Account" }} </h3>
+        </div>
+        <div class="modal-body">
+          <p>{{#tr this}}By deactivating your account, you will be logged out immediately.{{/tr}}</p>
+          <p>{{t "Note that any bots that you maintain will be disabled." }}</p>
+        </div>
+        <div class="modal-footer">
+          <button class="btn btn-default" data-dismiss="modal" aria-hidden="true">{{t "Cancel" }}</button>
+          <button class="button btn-danger" id="do_deactivate_user_button">{{t "Deactivate now" }}</button>
+        </div>
+      </div>
```

Comment:
So playing with this I ran into a weird bug where sometimes, when you click the button, the screen fades but the modal doesn't actually appear.  I think the reproducer is this:

* login as Iago 
* Make hamlet and admin
* Deactivate yourself
* re-login as Hamlet
* Reactivate Iago
* Try to deactivate yourself

Shift-reloading seems to make the problem go away and the modal show up again.